compiler/src/dotty/tools/dotc/transform/ReifyQuotes.scala

```diff
@@ -136,7 +136,7 @@ class ReifyQuotes extends MacroTransform {
       else {
         val trefs = importedTypes.toList
         val typeDefs = for (tref <- trefs) yield {
-          val tag = New(defn.QuotedTypeType.appliedTo(tref), Nil)
+          val tag = New(defn.QuotedTypeType.appliedTo(tref), Nil) // FIXME: should be an implicitly inferred defn.QuotedTypeType.appliedTo(tref)
```

Comment:
@odersky could you have a look at this one. I did not manage to use the `MacroTransformWithImplicits` and `inferImplicitArg` to get the implicit argument. I looked at b466e0afa6ea1908fcef8dc0fe2b30fa2887949a but it might be too outdated.